internal/xds/translator/testdata/out/xds-ir/http3.listeners.yaml

```diff
@@ -2,21 +2,18 @@
     socketAddress:
       address: 0.0.0.0
       portValue: 10443
-      protocol: UDP
```

Comment:
why did this yaml change ?

Reply:
The order of the quick and tcp listener changes.